src/uwtools/drivers/chgres_cube.py

```diff
@@ -35,25 +35,33 @@ def namelist_file(self):
             input_files.append(base_file)
         if update_values := namelist.get(STR.updatevalues):
             config_files = update_values["config"]
-            for k in ["mosaic_file_target_grid", "vcoord_file_target_grid"]:
+            for k in ["mosaic_file_target_grid", "varmap_file", "vcoord_file_target_grid"]:
                 input_files.append(config_files[k])
             for k in [
                 "atm_core_files_input_grid",
-                "atm_files_input_grid",
```

Comment:
Why is this one removed?

Reply:
It looks like it was lost in the diff block, I'll get it back in